relayer/chains/penumbra/tx.go

```diff
@@ -2233,3 +2233,9 @@ func (cc *PenumbraProvider) SendMessagesToMempool(ctx context.Context, msgs []pr
 	cc.log.Debug("Received response from sending messages", zap.Any("response", sendRsp), zap.Error(err))
 	return err
 }
+
+// MsgRegisterCounterpartyPayee creates an sdk.Msg to broadcast the counterparty address
+func (cc *PenumbraProvider) MsgRegisterCounterpartyPayee(portID, channelID, relayerAddr, counterpartyPayee string) (provider.RelayerMessage, error) {
+	msg := feetypes.NewMsgRegisterCounterpartyPayee(portID, channelID, relayerAddr, counterpartyPayee)
+	return NewPenumbraMessage(msg), nil
+}
```

Comment:
This isn't going to work on Penumbra so rather than trying to compose the msg can we change the implementation of this method to be 

```
//TODO implement me
panic("implement me")
```

similar to how it is in other unsupported features? That way if someone even tries this it becomes immediately clear what is going on